scenarios/change-data-capture/README.md

```diff
@@ -0,0 +1,176 @@
+# Pravega CDC Demo
+
+This page demonstrates how to preserve the CDC (change data capture) from MySQL into Pravega and consume them in Flink.
+
+> Change data capture (CDC) is used to determine and track the data that has changed so that actions can be taken using the changed data. It is often used in data-warehouse environments where we want have a copy or log of the existing data.
+
+We use Pravega as the streaming storage system which can preserve the continuous data stream, so the change data can be both written to and read from a durable, elastic and consistent place. Also, the debezium json format is used as the intermediate format when we transfer the data. You may have a look at [Pravega](https://cncf.pravega.io/) and [Debezium](https://debezium.io/) respectively at their homepage.
```

Comment:
You may have a look at *the* [Pravega](https://cncf.pravega.io/) and [Debezium](https://debezium.io/) *homepages*.